### ISSUE TITLE
My iphone lost network when replacing hosts file,

### ISSUE THREAD

**Issue by thinhnat0** [StevenBlack/hosts#617]
My iphone lost network when replacing hosts file,

**Comment by StevenBlack** [comment]
Hi @thinhnat0, how do you change the hosts file on an iPhone?

**Comment by Laicure** [comment]
@thinhnat0 
I did it too on my iPhone 6s, 10.3.3 Jailbroken phone: No internet.
Then I tried reducing the source: still no internet.
Then I searched through the internet, [some says it only accepts up to 3k lines](https://www.reddit.com/r/jailbreak/comments/7m7n7w/discussion_ios_hosts_file_limit/).

So I was like, "damn, just 3k?! Never mind...".

Revert to:
```
127.0.0.1 localhost
255.255.255.255 broadcasthost
::1 localhost
```

I just gave up on it. I don't need any other complex method just to replace the hosts file (I just sold my iPhone days ago hehehe).
